napari/layers/points/points.py

```diff
@@ -451,7 +450,7 @@ def data(self, data: np.ndarray):
         elif len(data) > cur_npoints:
             # If there are now more points, add the size and colors of the
             # new ones
-            with self.events.set_data.blocker():
+            with self.events.slice_data.blocker():
```

Comment:
Can you comment on why we need blockers here? I thought the whole point of the refactor was to not need blockers? Or is it because the refactor isn't actually done with the points layer?

Reply:
`Points` refactor not done yet - hopefully this will go when it is!!!!